nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-pubsub-processors/pom.xml

```diff
@@ -35,6 +35,10 @@
             <artifactId>nifi-utils</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-expression-language</artifactId>
```

Comment:
@JPercivall I agree, it does make sense to use EL subject-less functions.

EL functions can be used from Processor through interfaces provided by nifi-api. But I needed an implementation StandardPropertyValue class to evaluate EL from onAttributeModified method. I think we can not eval EL from String. Can we?